src/common/components/giftViews/giftSimpleView/giftSimpleView.tpl.html

```diff
@@ -0,0 +1,5 @@
+<h5 class="mt-- mb0">{{$ctrl.gift['amount'] | currency}} {{$ctrl.gift.rate.recurrence.interval}}</h5>
+<div ng-switch="$ctrl.gift.rate.recurrence.interval">
+  <small ng-switch-when="Monthly" translate>On the {{$ctrl.gift['recurring-day-of-month'] | ordinal}} day of each month</small>
+  <!-- TODO: Other intervals -->
```

Comment:
I left these blank for now since I believe @OzzieOrca is working on helper services to generate these.